2020-09-30 19:39:01.925 | INFO     | __main__:serve:85 - 启动服务: 192.168.1.102:53939
2020-09-30 19:39:01.929 | INFO     | __main__:serve:88 - 服务注册开始
2020-09-30 19:39:01.934 | INFO     | __main__:serve:95 - 服务注册成功
2020-09-30 19:40:40.214 | ERROR    | grpc._server:_call_behavior:435 - An error has been caught in function '_call_behavior', process 'MainProcess' (43240), thread 'ThreadPoolExecutor-0_0' (42172):
Traceback (most recent call last):

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Lib\threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x000001C5CCAE5F70>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 42172)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Lib\threading.py", line 932, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x000001C5CCAE5CA0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 42172)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 42172)>
    │    │        │    └ (<weakref at 0x000001C5CEB79270; to 'ThreadPoolExecutor' at 0x000001C5CE9EED00>, <_queue.SimpleQueue object at 0x000001C5CEA0...
    │    │        └ <Thread(ThreadPoolExecutor-0_0, started daemon 42172)>
    │    └ <function _worker at 0x000001C5CEA0F0D0>
    └ <Thread(ThreadPoolExecutor-0_0, started daemon 42172)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Lib\concurrent\futures\thread.py", line 80, in _worker
    work_item.run()
    │         └ <function _WorkItem.run at 0x000001C5CEA0F1F0>
    └ <concurrent.futures.thread._WorkItem object at 0x000001C5CEC266A0>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Lib\concurrent\futures\thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
             │    │   │    │       │    └ {}
             │    │   │    │       └ <concurrent.futures.thread._WorkItem object at 0x000001C5CEC266A0>
             │    │   │    └ [<grpc._cython.cygrpc.RequestCallEvent object at 0x000001C5CEC1FA40>, <grpc._server._RPCState object at 0x000001C5CEC26730>, ...
             │    │   └ <concurrent.futures.thread._WorkItem object at 0x000001C5CEC266A0>
             │    └ <function _unary_response_in_pool at 0x000001C5CEA39A60>
             └ <concurrent.futures.thread._WorkItem object at 0x000001C5CEC266A0>

  File "C:\Users\Administrator\Envs\mxshop_srv\lib\site-packages\grpc\_server.py", line 552, in _unary_response_in_pool
    response, proceed = _call_behavior(rpc_event, state, behavior,
                        │              │          │      └ <bound method GoodsServicer.GoodsList of <goods_srv.handler.goods.GoodsServicer object at 0x000001C5CEA176D0>>
                        │              │          └ <grpc._server._RPCState object at 0x000001C5CEC26730>
                        │              └ <grpc._cython.cygrpc.RequestCallEvent object at 0x000001C5CEC1FA40>
                        └ <function _call_behavior at 0x000001C5CEA39670>

> File "C:\Users\Administrator\Envs\mxshop_srv\lib\site-packages\grpc\_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
                           │        │         └ <grpc._server._Context object at 0x000001C5CEC268B0>
                           │        └ priceMin: 50
                           │          
                           └ <bound method GoodsServicer.GoodsList of <goods_srv.handler.goods.GoodsServicer object at 0x000001C5CEA176D0>>

  File "C:\Users\Administrator\PycharmProjects\mxshop_srvs\goods_srv\handler\goods.py", line 92, in GoodsList
    rsp.data.append(self.convert_model_to_message(good))
    │   │           │    │                        └ <Goods: 424>
    │   │           │    └ <function GoodsServicer.convert_model_to_message at 0x000001C5CDA7A040>
    │   │           └ <goods_srv.handler.goods.GoodsServicer object at 0x000001C5CEA176D0>
    │   └ <google.protobuf.internal.python_message._FieldProperty object at 0x000001C5CDA4ED10>
    └ <class 'goods_pb2.GoodsListResponse'>

AttributeError: '_FieldProperty' object has no attribute 'append'
